### DISCUSSION TITLE
Ability to open the same host multiple times

### DISCUSSION THREAD

**Discussion by gabriel70** [isontheline/pro.webssh.net#967] [Ideas]
I may just be overlooking this feature, but it would be nice to open the same host multiple times.  I often need to run multiple ansible playbooks from our shared ansible server, and I usually have 3 or 4 terminals of that host open to run the scripts one after the other for different groups of remote servers.

I’m migrating completely over to WebSSH from Termius on all my iOS devices, and this is the only feature I’ve discovered that I can’t figure out how to do, or is missing.  

Great product

**Comment by isontheline** [comment]
Hello @gabriel70 👋

Thank you so much for reaching me out 🙏

In order to launch multiple connections (same or different) you can use the following steps : 
1. Launch a connection 
2. Tap the top left button (icon as windows)
3. Choose "New Tab"
4. Launch a connection (SSH, SFTP)
5. Switch between connections by using the top left icon (windows)

Could make a try?

Have a nice day ☀️
